mani_skill/envs/scene.py

```diff
@@ -129,13 +129,15 @@ def create_physical_material(
     ):
         return physx.PhysxMaterial(static_friction, dynamic_friction, restitution)
 
-    def remove_actor(self, actor):
+    def remove_actor(self, actor: Actor):
         if physx.is_gpu_enabled():
             raise NotImplementedError(
                 "Cannot remove actors after creating them in GPU sim at the moment"
             )
         else:
-            self.sub_scenes[0].remove_entity(actor)
+            entities = [l for l in actor._objs]
```

Comment:
small nit, this can be improved by doing [self.sub_scenes[0].remove_entity(x) for x in actor._objs]

Reply:
The last commit fixed this, I also applied it for the following function that performed the same operation